### PR TITLE
gst-play: Add a check condition in print_all_stream_info

### PR DESCRIPTION
**Pull request by JimmyOhn**
I add a check condition for stream_list in print function.
In another print function, it also check whether it's null or not.

### CODE REVIEW

File: gst-play/gst-play.c [hunk 1]
```diff
@@ -195,9 +195,12 @@ static void
 print_all_stream_info (GstPlayerMediaInfo * media_info)
 {
   guint count = 0;
-  GList *l, *list;
+  GList *list = NULL, *l;
```
Comment (sdroege) [inline gst-play/gst-play.c]: No need to initialize it with NULL here. gst_player_media_info_get_stream_list() will always return something

**Comment by sdroege** [comment]
Looks mostly good, just a code style issue :) Please update

**Comment by JimmyOhn** [comment]
I attached the new commit for your comment.

**Comment by sdroege** [comment]
Thanks, merged

**Pull request closed without merging**